agc/asm.c

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
The anal and asm plugins have a lot of shared code that would be hard to interlink properly without a lot of fuss with the build system which I wanted to avoid. This way, all code is in one place. Also, I suppose if someone installs one of the plugins, that person probably wants both, as this target is pretty specific. I asked on IRC when I was deciding this bit and got told by @Maijin that this is an acceptable solution.